.github/workflows/build-test.yml

```diff
@@ -154,7 +154,8 @@ jobs:
     needs: changes
     if: ${{ (github.event_name != 'pull_request' && !failure() && !cancelled()) || contains(fromJSON(needs.changes.outputs.samples), 'java-grpc')}}
     with:
-      java: 17, 21
+      # Can't test with Java 21+ currently because of https://github.com/lightbend/ssl-config/issues/367
+      java: 17
```

Comment:
Need to disablethe Java 21 tests for grpc, because it requires an https port to open, but we can't do that in Java 21. When removing the `testserver.httpsport` property https://github.com/playframework/play-samples/blob/1ae950ff8a9266eff47372ca0de0b9814de5b551/play-java-grpc-example/build.sbt#L26 then you end up with exceptions like:
```
[error] Test example.myapp.helloworld.HelloFunctionalTest.worksWithAGrpcClient failed: java.lang.IllegalArgumentException: GrpcClientSettings requires a server endpoint with ssl, but none provided, took 1.109 sec
[error]     at play.grpc.testkit.JavaPekkoGrpcClientHelpers.lambda$grpcClientSettings$2(JavaPekkoGrpcClientHelpers.java:62)
[error]     at scala.Option.getOrElse(Option.scala:201)
[error]     at play.grpc.testkit.JavaPekkoGrpcClientHelpers.grpcClientSettings(JavaPekkoGrpcClientHelpers.java:60)
[error]     at play.grpc.testkit.JavaPekkoGrpcClientHelpers.grpcClientSettings(JavaPekkoGrpcClientHelpers.java:20)
[error]     at example.myapp.helloworld.HelloFunctionalTest.newGreeterServiceClient(HelloFunctionalTest.java:64)
[error]     at example.myapp.helloworld.HelloFunctionalTest.worksWithAGrpcClient(HelloFunctionalTest.java:86)
[error]     at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
[error]     at java.lang.reflect.Method.invoke(Method.java:580)
[error]     ...
```